lib/rucio/core/replica.py

```diff
@@ -3426,7 +3442,19 @@ def get_replicas_state(scope=None, name=None, *, session: "Session"):
 
 
 @read_session
-def get_suspicious_files(rse_expression, available_elsewhere, filter_=None, logger=logging.log, younger_than=5, nattempts=0, nattempts_exact=False, *, session: "Session", exclude_states=['B', 'R', 'D'], is_suspicious=False):
+def get_suspicious_files(
+    rse_expression: str,
+    available_elsewhere: int,
```

Comment:
In theory this could be more precisely set as `SuspiciousAvailability`, but the check on line 3484 expands the `SuspiciousAvailability` enum into a `list[int]`, so it expects an `int`. No change is needed here, maybe in future we can think of modifying the change on line 3484 so that we can annotate `available_elsewhere` as `SuspiciousAvailability`.